### ISSUE TITLE
Feature Request: Keyboard shortcut to select links

### ISSUE THREAD

**Issue by stephen-burrows** [leggett/simplify#558]
In the reading pane, it would be helpful to have a keyboard shortcut (Maybe g [forward] / G [backward]) to focus on links in an email so that pressing Enter would open the link in a new tab and Ctrl+Enter would open it without the browser switching to the new tab

**Comment by leggett** [comment]
Interesting... `g` already does something so I can't use that.

`Tab` essentially already does this. It just also highlights other things (Gmail buttons and what not). 

I could maybe add an option such that the `Tab` key skips Gmail buttons and just jumps between the links and attachments in an email. I also plan to add this to the Ctrl+K menu → pulling out all the links for the email you're in and letting you use your arrow keys or auto-complete on them to open them.

**Comment by stephen-burrows** [comment]
Yes, the tab would work. Ideally it would start at the first link currently on screen, to handle long emails
